### ISSUE TITLE
[Enhancement] Users shouldnt chose between new options to exclude URL query parameters

### ISSUE THREAD

**Issue by SW-Vincent** [piwik/piwik#22879]
Since last release, we can chose between a custom list, a recommended list and common session parameters to exclude. While having these options is fantastic, we should be able to cumulate them.

## Summary

Here is how the interface looks : 
![image](https://github.com/user-attachments/assets/3892253c-97ef-4996-8ba7-1eda70c1b4b0)
The issue is that in a lot of scenarios we'd want to exclude **both the recommanded parameters AND a custom selection**, while the selector only allows to select one of these options. 

## Your Environment

* Matomo Version: mostly cloud

**Comment by randy-innocraft** [comment]
Hi @SW-Vincent,
Thank you for bringing this to our attention and for your valuable input.
Your suggestion seems like a valuable enhancement to our product.
We will forward this to our Product team for review and future consideration.
If you have any additional details or questions, please feel free to share them here.

**Comment by ronak-innocraft** [comment]
@SW-Vincent Thanks for your feedback on our recent release for URL query param exclusions. Have you seen an option to append Matomo recommended PII when you have selected custom option? 
![image](https://github.com/user-attachments/assets/91803a4b-e348-4d98-859c-5897451dfd86)

**Comment by SW-Vincent** [comment]
> Have you seen an option to append Matomo recommended PII when you have selected custom option? 

Hi, i hadn't noticed indeed.
If the recommended list is updated, and i have added the recommended parameters beforehand, will it update my custom list ? I'm not sure it is dynamic.

**Comment by michalkleiner** [comment]
@SW-Vincent no, it's not dynamic in the way you think. The button simply adds the current recommended list into your custom list and deduplicates it. If Matomo updates the recommended list, it will be mentioned in the changelog. At the moment it's not expected that the recommended list will change often.

**Comment by mattab** [comment]
thanks for the suggestion. The current solution looks good enough, so will close this 👍
